server/routes/api/auth.js

```diff
@@ -1,6 +1,10 @@
 const express = require('express');
+
 const router = express.Router();
-const bcrypt = require('bcrypt');
+// const bcrypt = require('bcrypt');
```

Comment:
Do we still need this? If not, let's remove it.

Reply:
Removed.